### ISSUE TITLE
Size of 1u switch holders

### ISSUE THREAD

**Issue by 7t2** [nickcoutsos/dactyl-flatpacked#5]
![IMG_20220420_122309](https://user-images.githubusercontent.com/103980579/164134599-7491b141-b352-4275-bfa5-fc11a1935766.jpg)

The 1u switch holders have a very small distance of material in the corners and snap easily, I think it would definitely be nice to add a little bulk if at all possible.

**Comment by nickcoutsos** [comment]
Yeah, that's quite narrow. You can only make it so thick before you have to worry about interference. Are you comfortable with keys being spaced out further to accommodate this? It gets particularly hair at the ends of the finger cluster columns due to the curvature which is another drawback to the number row.

**Comment by 7t2** [comment]
![increasedHeight3](https://user-images.githubusercontent.com/103980579/164218642-0a65ad25-0e0d-4b06-9d0a-4bcf9041c252.png)


I don't think you need to extend the holders out in both directions, only one. This ways the keys can still be the same distance apart from each other. The left and right sides are supported by the acrylic below them, so they dont need to be thickened, but the top and bottom of the holders definitely need to be thickened because they overhang the two pieces of acrylic below them, and as a side effect of making them thicker vertically, the corners no longer have such a short distance of plastic holding the sides together. I hope that makes sense, the key holders would be longer up and down, but the cutout is still 14mm x 14mm

**Comment by nickcoutsos** [comment]
Oh, I see. Do you think that will be enough? It's an easy change to make, I think I misjudged how little space there was between the plates already. And I suppose we can get away with that cutout being small, it's just there to help with alignment.

**Comment by nickcoutsos** [comment]
So updating the profile for the switch plate was very simple and the cutouts from plate to plate look alright, but this may be an issue for the very ends of the column support where the plate sits.
<img width="724" alt="Screen Shot 2022-04-23 at 12 25 51 PM" src="https://user-images.githubusercontent.com/1165066/164914810-a50dc23d-d451-4d81-9fb6-716606572f81.png">

That little nub could very easily break off. It's not exactly integral, but I can imagine that the _way_ it breaks off could lead to further damage. There are some options for mitigating that:

1. Leave the nub off if its width falls under some threshold. No risk of damage, but less help with alignment when gluing the plate. If that's OK there's another question of whether or not it makes sense to leave the slot cutouts for that end of those plates. You'd get a maybe cleaner look but then you have to try and distinguish between these very subtly different pieces during assembly (which is already true but somewhat unavoidable).
2. Shorten the nub. The cutouts in the column supports extend from the bottom of the plate to the top but this isn't a specific requirement so much as... it just made sense to do it that way. If it doesn't extend as far up then there's less of a chance for it to snap but it won't look as nice.

I'm inclined to go with option 1 but I'm realizing that there's an extra weird case in the thumb cluster where the innermost column is a single 2u plate, meaning there would be nothing to help with alignment.

<img width="780" alt="Screen Shot 2022-04-23 at 12 46 59 PM" src="https://user-images.githubusercontent.com/1165066/164915642-037dfc9f-2499-4aa2-ab2d-57b0da2df9a2.png">

This suggests that my original fix should be a little more nuanced and only reduce that cutout for smaller plates where it's needed.

**Comment by nickcoutsos** [comment]
Hey, so, this is interesting. I was looking back at the photo of your and felt a little confused at the amount of spacing between plates which looked so different and I finally realized that I already made a change for this 😊 (https://github.com/nickcoutsos/dactyl-flatpacked/commit/9f45d5a1363d2c6f8088616c5dee332fc113eb30). Once again I'm sorry this wasn't available to you!

I'm going to keep this open and I think there's some value to the points discussed above. Even with the extra material these are still pretty dainty so anything that can help with strengthening the plates is worth exploring. I remember reading a long time ago that when cutting corners into acrylic it can be a good idea to cut a tiny circular corner so that there are no sharp edges that can crack from strain. I should look into that again.

**Comment by 7t2** [comment]
I do think thickening the top and bottom (but not the left and right sides) should be enough. There is plenty of support under the left and right side.

I think that having a small nub at the end is more useful than not, because it allows the person assembling the project to simply push the acrylic together to align everything perfectly, as seen in this photo
![pushdactyl](https://user-images.githubusercontent.com/103980579/165208281-a2492dff-222e-4484-a7b0-14505b117bab.jpg)

For the extra weird case, I think it would be best to adjust the support in a similar fashion to what I have drawn here
![dactyl_corner](https://user-images.githubusercontent.com/103980579/165208703-9bbd3298-ef6e-4f6e-88bc-6b53cb693e48.jpg)

The 1.5u and 2u switch holders, I think they're stucturally sound enough to stay the size they are. But all 1u switch holders definitely need the added material at the top and bottom.

And on that final comment about circular corners, I think by virtue of cutting acrylic with a laser, the corners are already pretty well rounded. If we identify weaknesses in the structure through heavy use, then we might want to think about implementing further crack resistance in those specific points. But until then, the lasered corners seem to do the job well
![cornerholes](https://user-images.githubusercontent.com/103980579/165210019-31672da1-a63e-4ec4-99a8-ad852369a80d.jpg)
